learning-outcomes.md

```diff
@@ -0,0 +1,38 @@
+# Week 2 Learning Outcomes
+
+## HTTP
+
+### Asynchronicity
+
+- [ ] **We can write code that executes asynchronously**
+- [ ] We can use callbacks to make our code non blocking
+
+### Fetch
+
+- [ ] **We can use the fetch method to make API requests and receive responses**
+- [ ] We can configure the options argument of the fetch method to make GET and POST requests
+- [ ] We can use an API key to bypass rate limit restrictions on certain APIs
+
+- [ ] **We can use the map and filter methods to display different data based on user input**
+- [ ] We can return filtered items from an array that match specified criteria
+- [ ] We can map over an array to create a new array containing new values
+
+- [ ] **We can manipulate the DOM using JavaScript**
+- [ ] We can use different selector methods to select DOM nodes
+- [ ] We can add and remove DOM nodes to change the appearance of our website
+- [ ] We can toggle the classes applied to DOM nodes to change their properties
+
+## Design
+
+- [ ] **We can use consistent layout and spacing**
+- [ ] We can implement readability margins to ensure a comfortable reading experience on different devices
```

Comment:
What does "readability margins" mean? Preventing the line-length from getting too long or stopping the text hitting the edge of the page? (Or both?)

Reply:
I meant preventing the line length from getting too long. I'll rephrase to make it clearer